src/ai/backend/storage/netapp/__init__.py

```diff
@@ -0,0 +1,215 @@
+from __future__ import annotations
+
+import asyncio
+import json
+from pathlib import Path
+from typing import FrozenSet
+from uuid import UUID
+
+from ai.backend.common.types import BinarySize, HardwareMetadata
+
+from ..abc import CAP_METRIC, CAP_VFOLDER
+from ..exception import ExecutionError
+from ..types import FSPerfMetric, FSUsage, VFolderCreationOptions
+from ..vfs import BaseVolume
+from .netappclient import NetAppClient
+from .quotamanager import QuotaManager
+
+
+async def read_file(loop: asyncio.AbstractEventLoop, filename: str) -> str:
+    def _read():
+        with open(filename, "r") as fr:
+            return fr.read()
+
+    return await loop.run_in_executor(None, lambda: _read())
+
+
+async def write_file(
+    loop: asyncio.AbstractEventLoop, filename: str, contents: str, perm="w"
+):
+    def _write():
+        with open(filename, perm) as fw:
+            fw.write(contents)
+
+    await loop.run_in_executor(None, lambda: _write())
```

Comment:
How about replacing these two functions with `aiofiles`?

Reply:
I re-wrote with aiofiles. But it seems backend.ai this version doesn't have aiofiles installed? Can't pass mypy checks.
"Library stubs not installed for "aiofiles" (or incompatible with Python 3.9)"

Should Storage-Proxy python requirements updated with aiofiles?